ui/client/components/graph/NodeDetailsContent.js

```diff
@@ -193,7 +235,15 @@ export class NodeDetailsContent extends React.Component {
               this.showOutputVar && this.createField("input", "Output", "outputVar", "outputVar", false, null)
             }
             {this.createReadonlyField("input", "Node type", "nodeType")}
-            {(this.state.editedNode.parameters || this.state.editedNode.ref.parameters).map((param, index) => {
+            {NodeUtils.nodeType(this.props.node) === 'Join' &&
```

Comment:
I considered it but I came to conclusion that It would cause 1) code duplication 2) complex templating logic which will prepare all common fields + can add some additional fields in the middle. Do you have some better approach?

Reply:
maybe you're right - let's leave it as it is for now, since we want to change UI for branches anyway...